### ISSUE TITLE
Lottie animations cause app crash and throw "PersistedOffset: is in an unexpected state" error

### ISSUE THREAD

**Issue by walik-walik** [FlutterFlow/flutterflow-issues#4778]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Using Lottie animations in FlutterFlow version 5.0.9 causes the Chrome web app to crash. A repeating error message—"Another exception was thrown: PersistedOffset: is in an unexpected state"—appears in the console. Additionally, attempting to replace one Lottie animation with another does not work, and no specific error message is provided for this issue.

### Expected Behavior

Lottie animations should load and display correctly without causing the Chrome web app to crash. Replacing one Lottie animation with another should work seamlessly, allowing for a smooth swap without errors or interruptions.

### Steps to Reproduce

1. Open an existing or new project in FlutterFlow version 5.0.9.
2. Add a Lottie animation widget to any screen in the project.
3. Attempt to preview or run the project as a web application in the Chrome browser.
4. Observe the console for the repeating error message: "Another exception was thrown: PersistedOffset: is in an unexpected state."
5. Next, try replacing the initial Lottie animation with a different Lottie file in the widget’s properties.
6. Notice that the replacement animation does not load or display as expected, and there is no specific error message indicating why the swap failed.

**Note:** The issue appears specific to FlutterFlow 5.0.9 and affects only the Chrome web preview, as other environments may behave differently.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4KlM/l34xOtbxZ1s7fbcdFmikXJmY6UIIKlutuURMaF4PkG7YuQs+lVBJtXdOQTFZiPVqWmnsypq3ZudrtFPoVE0+ub4dq3bt5WA7jUk6uMp/RDZeaXnFTJ59nFXGO5JiJpAx9FPFfLXw6wV+YFt+/VgXaHqbjDFcNBfmHBJLMnlKGKyLjVxFJ5CdecCfn

### Visual documentation

https://github.com/user-attachments/assets/fff052be-be8b-4fb4-990b-bc507ecb1ec2

https://github.com/user-attachments/assets/9cafb4c3-305e-4bb0-8b75-d5572941d2aa

![Bildschirmfoto 2024-10-30 um 10 58 50](https://github.com/user-attachments/assets/97e6e824-71ec-474f-9cb4-85c8cc2b340b)

### Environment

```markdown
- FlutterFlow version: 5.0.9
- Platform: We and MacOS Desktop
- Browser name and version: Chrome Version 130.0.6723.71 
- Operating system and version affected: macOS Sequia 15.0.1 (24A348)
```

**Comment by Alezanello** [comment]
Hello!

I wasn’t able to replicate the issue on my end. However, I’ve shared a public project below where I attempted to replicate it. Feel free to clone, modify, and test it as needed. Please let me know if there’s anything specific I might have missed!<br><br>[https://app.flutterflow.io/project/qa-2927-4you4q](https://app.flutterflow.io/project/qa-2927-4you4q)

**Comment by walik-walik** [comment]
Hello!

Thank you for sharing the public project for testing. I’ve cloned and tested it on my end, and I am still experiencing the same issue. When running the app in the Chrome browser, it crashes, and I see the "PersistedOffset: is in an unexpected state" error looping in the console.

Additionally, I receive the following warning message in FlutterFlow:
"Warning: Box Fit and Frame Rate properties may not be working properly. Try reentering the Network Path or reselecting the Asset to fix this."

In the FlutterFlow editor, the Lottie animation displays correctly. However, in the Chrome browser, the animation does not display, and the error appears in a loop as mentioned.

To help with diagnosis, I’m also attaching the JSON file for the Lottie animation I used. The error occurs with this file as well as with other Lottie animations, and I am also unable to select new animations after encountering the error.

Please let me know if there are any additional steps or configurations you recommend. I’m happy to try any suggestions that might help isolate the cause.

Thanks again for your assistance!
[Trophy.json](https://github.com/user-attachments/files/17619305/Trophy.json)

**Comment by walik-walik** [comment]
I noticed that the error seems to occur specifically when using Local Run. When the app is hosted normally, the Lottie animations work fine without any issues. However, during Local Run in the Chrome browser, the app crashes, and I see the "PersistedOffset: is in an unexpected state" error looping in the console.

Since it looks like you weren't able to reproduce the issue before, could you please try testing it with Local Run? This might help in identifying the cause of the problem.

Thank you for your assistance!

**Comment by walik-walik** [comment]
> This issue is stale because it has been open for 7 days with no activity. If there are no further updates, a team member will close the issue.

Yes, I provided updates last week—please check my previous comments above. The issue persists, specifically when using Local Run, as I mentioned in my latest update.

Thank you for looking into this!

**Comment by rzambroni** [comment]
Hey @walik-walik,

Apologies for the delay! I also tested the blank project @azanello provided, and it’s working fine on my end, even when using Local-run and running it on Chrome.

Are you still experiencing this issue? I know there were some fixes made recently related to Lottie animations—nothing directly tied to this, but it’s possible it was resolved indirectly.

Looking forward to your feedback!
